src/components/storyUtils/SelectShowcase/MultiSelectShowcase.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react';
+
+import Select from '../../Select';
+import { SelectOption } from '../../Select/Select';
+import { container, wrapper } from './MultiSelectShowcase.style';
+
+type Props = {
+  minCharactersToSearch?: number;
+};
+
+const options = [
+  { value: 'chocolate', label: 'Chocolate' },
+  { value: 'strawberry', label: 'Strawberry' },
+  { value: 'banana', label: 'Banana' },
+  { value: 'citrus', label: 'Citrus' },
+  { value: 'sorbet', label: 'Sorbet' },
+  { value: 'caramel', label: 'Caramel' },
+  { value: 'cookie_dough', label: 'Cookie Dough' },
+  { value: 'buttered_pecan', label: 'Buttered Pecan' },
+  { value: 'vanilla', label: 'Vanilla' },
+  { value: 'garlic', label: 'Garlic', isDisabled: true },
+];
+
+const selectedOptions = [
+  { value: 'strawberry', label: 'Strawberry' },
+  { value: 'banana', label: 'Banana' },
+  { value: 'citrus', label: 'Citrus' },
+  { value: 'sorbet', label: 'Sorbet' },
+  { value: 'cookie_dough', label: 'Cookie Dough' },
+  { value: 'buttered_pecan', label: 'Buttered Pecan' },
+];
+
+export const dummyUnrefinedData = new Array(15).fill(undefined).map((__, index) => ({
+  value: index,
+  label: `Test option ${index}`,
+}));
+
+const SelectShowcase: React.FC<Props> = ({ minCharactersToSearch = 0 }) => {
+  const [asyncOptions, setAsyncOptions] = useState<SelectOption[]>(dummyUnrefinedData);
+  const [isLoading, setIsLoading] = React.useState(false);
+
+  const mockedApiCall = (term: string) => {
+    new Promise<SelectOption[]>((resolve) => {
+      setTimeout(() => {
+        resolve(dummyUnrefinedData);
+        setIsLoading(false);
+      }, 1500);
+    }).then((values) => {
+      const filteredValues = values.filter((option) => option.label.includes(term));
+      setAsyncOptions(filteredValues);
+    });
+  };
+
+  return (
+    <div>
+      <h4>Multiselect with simple Search</h4>
```

Comment:
For some reason this h4 element seems to lose their styles on render 🤔 

![Screenshot_42](https://user-images.githubusercontent.com/57801353/227490902-87e674f7-7236-41cc-90f7-946b56b86ce4.png)